diffpy/snmf/correlation.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def pearson(data):
+    """
+
+    Parameters
+    ----------
+    data
+
+    Returns
+    -------
+
+    """
+    data = np.asarray(data)
+    x = data[:, 0]
```

Comment:
This is probably too specific to put inside the function.  Just take some simple data type and keep the reshaping of the data on the outside of the function.

Reply:
Is it better to index data directly when I call np.corrcoef and get rid of x and y?

Reply:
I am not sure I understand the question.   It will depend on the use-case.